src/lib/db/feature-toggle-client-store.ts

```diff
@@ -238,6 +245,16 @@ export default class FeatureToggleClientStore
         });
     }
 
+    private static removeTitlesFromStrategies(
+        features: IFeatureToggleClient[],
+    ) {
+        features.forEach((feature) => {
+            feature.strategies.forEach((strategy) => {
+                delete strategy.title;
```

Comment:
JS engines like V8 don't like delete as it affects perf. What if we iterated over features with map by omiting what is not needed and combining removeTitlesFromStrategies with removeIdsFromStrategies to avoid re-iterating

Reply:
Ah, I wasn't aware of that. But yeah, that seems like a fine suggestion. I left the simplest case in here for now assuming that the number of features is usually insignificant, but avoiding an iteration is better if we can. I'll have a look!